test/e2e/resource_fairness/resource_fairness_test.go

```diff
@@ -24,36 +24,116 @@ import (
 	"time"
 
 	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
 
 	"github.com/apache/yunikorn-core/pkg/common/configs"
+	"github.com/apache/yunikorn-core/pkg/webservice/dao"
 	"github.com/apache/yunikorn-k8shim/pkg/common/constants"
 	tests "github.com/apache/yunikorn-k8shim/test/e2e"
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/common"
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/k8s"
 	"github.com/apache/yunikorn-k8shim/test/e2e/framework/helpers/yunikorn"
 )
 
+var kClient k8s.KubeCtl
+var restClient yunikorn.RClient
+var nodes *v1.NodeList
+var workerID1 string
+var workerID2 string
+var namespace *v1.Namespace
+var err error
+var queuePath string
+var basicUtiliz float64
```

Comment:
Nit: use the full word in the variable name, eg. `basicUtilization`. This sounds a bit strange to me.